signer/src/stacks/api.rs

```diff
@@ -1574,7 +1504,84 @@ fn extract_signatures_required(value: Value) -> Result<Option<u16>, Error> {
     }
 }
 
+#[derive(Debug, Deserialize)]
+struct GetTenureHeadersLightApiResponse {
+    #[serde(rename = "consensus_hash")]
+    pub _consensus_hash: String,
+    pub burn_block_height: u64,
+    pub burn_block_hash: String,
+    pub stacks_blocks: Vec<GetTenureHeadersLightApiStacksBlock>,
+}
+
+#[derive(Debug, Deserialize)]
+struct GetTenureHeadersLightApiStacksBlock {
+    pub block_id: String,
+    #[serde(rename = "header_type")]
+    pub _header_type: String,
+    #[serde(rename = "block_hash")]
+    pub _block_hash: String,
+    pub parent_block_id: String,
+    pub height: u64,
+}
```

Comment:
Even though these structs are private, we should add comments to the structs themselves and their fields. Also, could you please add a link to the relevant structs on stacks core?